File: app/src/main/java/com/alexpaxom/workrhythm/remindernotification/ReminderNotificationReceiver.java
package com.alexpaxom.workrhythm.remindernotification;

import android.content.BroadcastReceiver;
import android.content.Context;
import android.content.Intent;


public class ReminderNotificationReceiver extends BroadcastReceiver {

    @Override
    public void onReceive(Context context, Intent intent) {
        NotificationFactory notificationFactory = new NotificationFactory(context);
        notificationFactory.notificationManager.notify(NotificationFactory.REMIND_NOTIFICATION_ID, notificationFactory.getReminderNotification());
    }
}

File: app/src/main/java/com/alexpaxom/workrhythm/app/remindernotification/ReminderNotificationReceiver.kt
package com.alexpaxom.workrhythm.app.remindernotification


import android.content.BroadcastReceiver
import android.content.Context
import android.content.Intent

class ReminderNotificationReceiver : BroadcastReceiver() {
    override fun onReceive(context: Context, intent: Intent) {
        val notificationFactory = NotificationFactory(context)
        notificationFactory.notificationManager.notify(
            NotificationFactory.REMIND_NOTIFICATION_ID,
            notificationFactory.reminderNotification
        )
    }
}